frontend/src/components/BaseDashboard/components/DashboardSidebar/index.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import "./DashboardSidebar.scss";
+import dashboard_routes from "../../dashboard-routes";
+import { NavLink } from "react-router-dom";
+import classNames from "classnames";
+
+const DashboardSidebar = () => {
+  const instrumentsItems = dashboard_routes.map((item, index) => (
+    <NavLink
+      exact={true}
+      key={index}
+      to={item.path}
+      activeClassName="dashboardActiveLink"
+      className={classNames(item.className, "dashboardLink")}
```

Comment:
```suggestion
      className={[item.className, "dashboardLink"].join(" ")}
```
Are there any advantages using a library for this?